### ISSUE TITLE
Add an ability to store EMR output to HDFS.

### ISSUE THREAD

**Issue by ashkop** [Yelp/mrjob#887]
I need to save output of EMR job to HDFS because I have to enable SSE on S3 for output and this can be done only with S3DistCp. Or maybe it will be better to allow adding S3DistCp as a step and then intermediate results will be stored to HDFS as usual?

**Comment by davidmarin** [comment]
No, probably better to let you specify a path on HDFS, since there isn't control over where the intermediate results go.

What happens now if you try to specify an HDFS URI with `--output-dir`?
